### PR TITLE
Implementing custom_request_headers 

### PR DESCRIPTION
**Pull request by femrtnz**
Hi All,

I'm adding the custom request headers into the module as implemented on those links below.

https://cloud.google.com/load-balancing/docs/custom-headers#variables
https://www.terraform.io/docs/providers/google/r/compute_backend_service.html

Please let me know if you have any questions!

### CODE REVIEW

**Comment by femrtnz** [comment]
If someone could point out what is failing I'm happy to fix it :)

File: examples/multi-backend-multi-mig-bucket-https-lb/main.tf [hunk 1]
```diff
@@ -147,6 +147,8 @@ module "gce-lb-https" {
       enable_cdn                      = false
       session_affinity                = null
       affinity_cookie_ttl_sec         = null
+      custom_request_headers          = null
+
       health_check                    = local.health_check
```
Comment (morgante) [inline examples/multi-backend-multi-mig-bucket-https-lb/main.tf:152]: Please run `terraform fmt`.

```suggestion
      health_check = local.health_check
```

**Review by morgante — CHANGES_REQUESTED** [review]
Thanks for the contribution.

You'll need to add `custom_request_headers = null` in all the existing examples.

File: examples/multi-backend-multi-mig-bucket-https-lb/main.tf (hunk 1 above)
Comment (femrtnz) [reply examples/multi-backend-multi-mig-bucket-https-lb/main.tf]: Thanks!

**Comment by femrtnz** [comment]
> Thanks for the contribution.
> 
> You'll need to add `custom_request_headers = null` in all the existing examples.

Hi @morgante thanks for the message. Please let me know if there are anything else to change.

**Comment by morgante** [comment]
@femrtnz Looks like you missed one example: terraform_validate ./examples/https-redirect

**Comment by femrtnz** [comment]
> @femrtnz Looks like you missed one example: terraform_validate ./examples/https-redirect

hi @morgante 

Actually I add this already here: https://github.com/terraform-google-modules/terraform-google-lb-http/pull/122/files#diff-61e9ef05efa9836976128999f5f1ddbbL114

Am I missing something?

**Comment by morgante** [comment]
@femrtnz You also need to add `iap_config = null` to that module. It looks like we missed it earlier.

**Comment by femrtnz** [comment]
> @femrtnz You also need to add `iap_config = null` to that module. It looks like we missed it earlier.

Hey @morgante , sure thing. Just pushed the missing field.

Thanks

**Comment by morgante** [comment]
@femrtnz Linter is still failing. Could you please test locally and get things working? We explain how here: https://github.com/terraform-google-modules/terraform-google-lb-http/blob/master/CONTRIBUTING.md

**Comment by femrtnz** [comment]
Hi @morgante Just fixed the Linter :)

Please let me know if you have any questions.

Thanks

**Review by morgante — APPROVED** [review]

**Comment by naseemkullah** [comment]
@morgante can we cut a release with this feature please?

**Pull request merged**